src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginCache.cs

```diff
@@ -501,55 +486,14 @@ private void SetupAssemblyResolver(string extensionAssembly)
             // Add assembly resolver which can resolve the extensions from the specified directory.
             if (this.assemblyResolver == null)
             {
-                this.assemblyResolver = new AssemblyResolver(resolutionPaths);
+                this.assemblyResolver = new AssemblyResolver(resolutionPaths, EqtTrace.PlatformTrace);
```

Comment:
AssemblyResolver just requires a way to `LogMessage(level, message)`. Why pass the platform trace to it?